catalan.ini

```
@@ -0,0 +1,669 @@
+﻿[Header]
+Language = "Català"
+Author = "Minepika55"

```

Opcional, but seeing the changes...

```suggestion
Author = "Minepika55, GuillemRoca"
```